server/templates/tools/map_examples.json

```diff
@@ -9,16 +9,6 @@
       "text": "districts in India",
       "url": "#%26sv%3DDifferenceRelativeToBaseDate2006_Max_Temperature_RCP45%26pc%3D0%26pd%3Dcountry%2FIND%26ept%3DAdministrativeArea2"
     }]
-  }, {
-    "header": "Annual Non-Biogenic Carbon Dioxide Emissions",
```

Comment:
Do we know why these examples don't work anymore? Is it related to data changes or page layout changes?

Reply:
I believe it's due to data changes; see chat thread.

Reply:
Got it. It'd be good to do a follow-up PR with replacements that work.